models/amazon_ads.yml

```diff
@@ -275,13 +275,13 @@ models:
       - name: cost
         description: "{{ doc('cost') }}"
 
-  - name: amazon_ads__search_report
-    description: Each record in this table represents the daily performance at the search term level.
-    tests:     
-      - dbt_utils.unique_combination_of_columns:
-          combination_of_columns:
-            - date_day
-            - search_term
+  - name: amazon_ads__search_report 
+    description: Each record in this table represents the daily performance at the search term level. 
+    tests: 
+      - dbt_utils.unique_combination_of_columns: 
+          combination_of_columns: 
+            - date_day 
+            - search_term 
```

Comment:
I am seeing a test failure on this model.
![image](https://user-images.githubusercontent.com/74217849/211377321-e1f37886-676f-4687-a52b-aa3dd6577ed8.png)


Are you seeing this failure as well? Why do you think this may be erroring out on my end. Do we possibly need to adjust the test for this model?

Reply:
I am also now getting this error--I think when I was developing I might have been running the tests on the seed data, so it didn't get caught (got my dev and integration_test crossed). I reviewed the data and saw I need to add `keyword_id` column to the test, which passes now.